src/hooks/fetch.test.ts

```diff
@@ -3,6 +3,10 @@ import fetchMock from 'jest-fetch-mock';
 import { FetchInstrumentation } from './fetch';
 import { SpansContainer } from '../globals';
 
+// Note: With the latest version of axios, using jest-fetch-mock can cause some tests to fail.
+// To work around this issue, we've moved those specific tests to a separate file (fetchUtils.test.ts).
```

Comment:
Specify that the parsing of headers is effected

Reply:
Done